.ci/travis/travis.sh

```diff
@@ -171,6 +171,21 @@ no-exception-test-alot-of-project1)
   groovy ./launch.groovy --listOfProjects projects-for-travis.properties --config checks-nonjavadoc-error.xml
   ;;
 
+cobertura-check)
+  mvn clean compile cobertura:check cobertura:cobertura
+  grep -Po "<class name=\".+\"" target/site/cobertura/coverage.xml | grep -Po "com.puppycrawl.tools.checkstyle\.?.*\" filename" | sed 's/\" filename//g' | sed 's/\./\//g' | sort | uniq > cobertura_classes.log
```

Comment:
It is ok, but ...
I do not recommend to use grep for xml parsing, it is not stable solution
Consider to use xmlstarlet, youcan find example of usages in this repo and sevnu.